src/keria/app/agenting.py

```diff
@@ -231,9 +232,25 @@ def delete(self, agent):
 
         del self.agents[agent.caid]
 
+    def close(self, agent):
+        agent.hby.ks.close(clear=False)
+        agent.seeker.close(clear=False)
+        agent.exnseeker.close(clear=False)
+        agent.monitor.opr.close(clear=False)
+        agent.notifier.noter.close(clear=False)
+        agent.rep.mbx.close(clear=False)
+        agent.registrar.rgy.close()
+        agent.mgr.rb.close(clear=False)
+        agent.hby.close(clear=False)
+        self.remove(agent.doers)
```

Comment:
Maybe safer to remove these first, in case one of the doers tries to read from the DB while closing